### ISSUE TITLE
@import as reference (without css output) like less

### ISSUE THREAD

**Issue by jerchap** [sass/sass#2608]
I'd like to use the @import directive so I an extend classes, but without the bloat of output the css of what I'm importing.  I only want the css output to be what I extend.

**Comment by nex3** [comment]
This has been brought up before in the context of the module system, but I don't think it has its own issue yet so I guess this one will do.

For context: this is something I think is worth investigating, but probably not until the current module system work is done.

**Comment by MathijsvVelde** [comment]
+1

**Comment by nex3** [comment]
Marking this as a duplicate of #2791 (since that explicitly refers to `@use`).
